src/test/java/teammates/test/cases/browsertests/InstructorFeedbackPageUiTest.java

```diff
@@ -721,80 +729,79 @@ public void testDatePickerScripts() {
         Calendar initialCal = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
         initialCal.setTime(cal.getTime());
         
-        cal.add(Calendar.DATE, 30);
-        feedbackPage.fillTimeValueForDatePickerTest(Const.ParamsNames.FEEDBACK_SESSION_STARTDATE, cal);
+        Calendar changedStartDate = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        changedStartDate.set(2014, 4, 16, 0, 0, 0);
+        feedbackPage.fillTimeValueForDatePickerTest(Const.ParamsNames.FEEDBACK_SESSION_STARTDATE, changedStartDate);
         
-        assertEquals(sdf.format(initialCal.getTime()),
-                     feedbackPage.getValueOfDate(Const.ParamsNames.FEEDBACK_SESSION_ENDDATE));
+        valueOfEndDate = feedbackPage.getValueOfDate(Const.ParamsNames.FEEDBACK_SESSION_ENDDATE);
+        assertEquals(sdf.format(initialCal.getTime()), valueOfEndDate);
         
         
         ______TS("decreasing start date affects visible time, end date range and publish date range");
         
-        cal.add(Calendar.DATE, -35);
-        feedbackPage.fillTimeValueForDatePickerTest(Const.ParamsNames.FEEDBACK_SESSION_STARTDATE, cal);
+        changedStartDate.add(Calendar.DATE, -35);
```

Comment:
- `decreasedStartDate` (i.e. define a new variable)
- Set to a specific date instead of relying on a base date and rewinding 35 days.